### PR TITLE
fix(coordinator): cleanup session key service and use plaintext approvals

### PR DESCRIPTION
**Pull request by ctrlc03**
Cleanup session key service

### CODE REVIEW

File: packages/coordinator/ts/common/networks.ts [hunk 1]
```diff
@@ -1,64 +1,84 @@
+import {
```
Comment (kittybest) [inline packages/coordinator/ts/common/networks.ts:1]: just thinking about the frontend interface would also need these information, maybe put this to a place that both interface and coordinator can call? (no need to do in this PR)

File: packages/coordinator/ts/sessionKeys/sessionKeys.service.ts [hunk 2]
```diff
@@ -64,26 +62,22 @@ export class SessionKeysService {
...
    * @param chain - the chain to use
-   * @returns
+   * @returns a KernelAccountClient
    */
   async generateClientFromSessionKey(
     sessionKeyAddress: Hex,
-    encryptedApproval: string,
-    chain: Chain,
+    approval: string,
+    chain: ESupportedNetworks,
```
Comment (kittybest) [inline packages/coordinator/ts/sessionKeys/sessionKeys.service.ts:72]: why we don't need to encrypt the approval now?

**Review by kittybest — COMMENTED** [review]
Thanks!

File: packages/coordinator/ts/common/networks.ts (hunk 1 above)
Comment (ctrlc03) [reply packages/coordinator/ts/common/networks.ts:1]: good point, maybe maci-cli/sdk?

File: packages/coordinator/ts/sessionKeys/sessionKeys.service.ts (hunk 2 above)
Comment (ctrlc03) [reply packages/coordinator/ts/sessionKeys/sessionKeys.service.ts:72]: so technically the approval is just a string, without the session key private key one cannot do anything. So even if the approval is stolen they cannot do anything, and the temporary private key never leaves the server, so if they had access to the private key, they would also be able to get the approval whether it's encrypted or not

**Review by kittybest — APPROVED** [review]
Thanks!

**Pull request merged**